Gopkg.toml

```diff
@@ -126,3 +126,8 @@ ignored = [
 [[constraint]]
   branch = "master"
   name = "github.com/kube-object-storage/lib-bucket-provisioner"
+
+# workaround until fix is merged https://github.com/kube-object-storage/lib-bucket-provisioner/pull/136
+[[override]]
+  name = "github.com/kube-object-storage/lib-bucket-provisioner"
+  source = "github.com/rohantmp/lib-bucket-provisioner"
```

Comment:
Please note: this is my personal fork. The only changes on it are bumping the kube version in Gopkg.toml to 14.1

I also checked that it was building successfully.